### PR TITLE
Fix deprecated Buffer construction

### LINKED ISSUE #10: [DEP0005] DeprecationWarning: Buffer() is deprecated…
```text
$ nodejs -e 'require("nofs").writeFile("/dev/null", "")' |& fmt
(node:24616) [DEP0005] DeprecationWarning: Buffer() is deprecated
due to security and usability issues. Please use the Buffer.alloc(),
Buffer.allocUnsafe(), or Buffer.from() methods instead.

$ nodever nofs
require('nofs/package.json').version = 0.11.22
Node.js v12.1.0, npm v6.9.0, Ubuntu 16.04.6 LTS xenial,
```

### PR DESCRIPTION
**Pull request by mk-pmb**
Fixes #10

### CODE REVIEW

**Comment by ysmood** [comment]
@mk-pmb Thank you, could you please remove this line too, so that the CI can pass.

https://github.com/ysmood/nofs/blob/6d18284ffabcaf117dd575a8a9b0b4e9a587c2c2/.travis.yml#L4

**Comment by ysmood** [comment]
Seems like it's time to drop support for nodejs v0.11 now.

**Comment by mk-pmb** [comment]
Here you go!

File: src/main.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ npath = require('./path');
 
 child_process = require('child_process');
 
+function encodeNonBuffer(data, encoding) {
```
Comment (ysmood) [inline src/main.js]: Won't it better to put this function into the `src/utils.js` file?

**Review by ysmood — CHANGES_REQUESTED** [review]
Looks goods, just one fix needed unless you have a good reason

File: src/main.js (hunk 1 above)
Comment (mk-pmb) [reply src/main.js]: Probably would be better. I'll update the PR.

**Comment by ysmood** [comment]
@mk-pmb nofs@0.12.0 published on npm

**Comment by mk-pmb** [comment]
Thanks!

**Pull request merged**